app/components/samples/table_component.html.erb

```diff
@@ -20,28 +20,25 @@
               classes: class_names('px-3 py-3', 'sticky left-0 min-w-56 max-w-56 z-10 bg-slate-50 dark:bg-slate-700': index.zero?, 'sticky left-56 z-10 bg-slate-50 dark:bg-slate-700': index == 1)
             ) do %>
               <% if index.zero? and @abilities[:select_samples] %>
-                <%= search_form_for(
-                  @q,
-                  url: select_samples_url(**request.query_parameters),
-                  html: { "data-controller": "filters", id: "select-all-form" },
-                  class: "filters align-middle float-left"
-                ) do |f| %>
-                  <input type="hidden" name="format" value="turbo_stream"/>
-
-                  <label for="select-all" class="sr-only"><%= t(:".select_all") %></label>
+                <div
```

Comment:
**nitpick (blocking):** Can you simplify this to just a checkbox tag. See below:

```
<%= check_box_tag "select-page",
                title: t(:".select_page"),
                "aria-label": t(:".select_page"),
                data: {
                  action: "input->selection#togglePage",
                  controller: "filters",
                  selection_target: "selectPage",
                },
                class:
                  "w-4 h-4 mr-2.5 text-primary-600 bg-slate-100 border-slate-300 rounded focus:ring-primary-500 dark:focus:ring-primary-600 dark:ring-offset-slate-800 focus:ring-2 dark:bg-slate-700 dark:border-slate-600" %>
```

This change will required the `Ransack::SortComponent` to be updated, specifically the `flex` on line 1 of the html will need to be changed to `inline-flex`.

Additionally `select_all` translation needs removed from `table_component.en.yml` and `table_component.fr.yml` and `select_page` will need to be added.

Reply:
The translations for the table component are stored in `app/components/samples/table_component.en.yml` and `app/components/samples/table_component.fr.yml`. Please make your changes there.

Reply:
cca65f5eeb651e901886f699275fed3aa4e8a6f3